temba/msgs/models.py

```diff
@@ -831,9 +835,21 @@ def process_message(cls, msg):
 
         cls.mark_handled(msg)
 
+        # Chatbase parameters to track logs
+        chatbase_not_handled = True
+
         # if this is an inbox message, increment our unread inbox count
         if msg.msg_type == INBOX:
             msg.org.increment_unread_msg_count(UNREAD_INBOX_MSGS)
+        elif msg.msg_type == FLOW:
+            chatbase_not_handled = False
+
+        # Sending data to Chatbase API
+        org = msg.org
+        if org.is_connected_to_chatbase():
```

Comment:
`is_connected_to_chatbase` calls `config_json()` and then it's immediately called again. Maybe instead of having a `is_connected_to_chatbase` method, you could have a `get_chatbase_credentials` which returns the key/version or None - that way you can use it to both 1) determine if org is configured for chatbase, and 2) get the credentials